alea/include/alps/alea/autocorr.hpp

```diff
@@ -154,6 +154,9 @@ class autocorr_result
     /** Returns sample size, i.e., number of accumulated data points */
     size_t count() const { return level_[0].count(); }
 
+    /** Return sum of squared sizes */
+    double count2() const { return level_[find_level(DEFAULT_MIN_SAMPLES)].count(); }
```

Comment:
Not sure if `count2()` is a good name. Maybe, `count_squared()` or something? How important is it to keep the name short?

Reply:
I neither find `count2` or `count_squared` to be particularly descriptive. (It's the sum of squared weights).  For consistency with `data2`, I therefore named it `count2`.